src/MAMCTS.jl

```diff
@@ -33,10 +34,21 @@ function POMDPs.simulate(sim::RolloutSimulator, mdp::JointMDP, policy::Policy, i
 
     s = initialstate
 
-    disc = 1.0
-    r_total = zeros(n_agents(mdp))
-    step = 1
+    
+    # TODO: doesn't this add unnecessary action search?
+    r = @gen(:r)(mdp, s, action(policy, s), sim.rng)
```

Comment:
Eh under the circumstances it is the least bad option, I think.

Reply:
I mean, I could do a random action as well?